config.py

```diff
@@ -32,7 +32,7 @@ class Config(object):
     The base configuration option. Contains the defaults.
     """
 
-    DEBUG = False
+    DEBUG = True
```

Comment:
Yeah, was using it for my purposes, got lost in the large diff. Resolved it, Thanks.